game/scripts/vscripts/components/progression/hero_progression.lua

```diff
@@ -5,96 +5,114 @@ if HeroProgression == nil then
     ChatCommand:LinkCommand("-levelup", "OnLevelUpChatCmd", HeroProgression)
 end
 
+GameEvents:OnPlayerLevelUp(function(keys)
```

Comment:
A console command. Reloads Lua scripts so that at least some things can be changed and tested without having to restart the whole map.